rand_distr/src/utils.rs

```diff
@@ -9,10 +9,10 @@
 //! Math helper functions
 
 use crate::ziggurat_tables;
-use core::{cmp, ops};
 use rand::distributions::hidden_export::IntoFloat;
 use rand::Rng;
 
+/*
```

Comment:
This is WIP I take it? But why modify the `Float` trait instead of simply deleting it?

Reply:
Ah, yes. I forgot to remove it. Also I wonder that to do with the Poisson impl, should I delete it as well?